docs/install_linux.md

```diff
@@ -22,20 +22,22 @@ You will be asked for a password when doing the command, that's all that really
 **Keep this in mind when setting up, so you don't waste our time with errors that stem from lacking permissions!**
 While you may think running the following commands as the root user is easy, we advice against it due to the security risks involved when using such a powerful user, instead please use `sudo`.
 ##Installing Node
-First off, we'll install Node.JS, the runtime WildBeast uses. The current 6.x.x version works fine.
+First off, we'll install Node.JS, the runtime WildBeast uses, with [n-install](https://github.com/mklement0/n-install) which installs [n](https://github.com/tj/n). ***Note:*** n installs node *per user* so if you have more than one user on your machine, they must also install n.
```

Comment:
Done in [8dfefbb](https://github.com/TheSharks/WildBeastDocs/pull/20/commits/8dfefbb0771518ab218dfcdd5dbe6db8a8f57b94)